src/build-data/makefile.in

```diff
@@ -40,18 +41,18 @@ libs: $(LIBRARIES)
 
 # Misc targets
 
+docs:
+%{build_doc_commands}
```

Comment:
I suggest to have docs.stamp or any other file created, so that at make install the sphinx will not run again

Reply:
Good idea thanks